react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -113,7 +110,11 @@ const ResourceAllocationFormItems: React.FC<
   const [isPendingAgentList, startAgentListTransition] = useTransition();
 
   const currentProject = useCurrentProjectValue();
-  const currentResourceGroup = useCurrentResourceGroupValue(); // use global state
+  const currentResourceGroupInFrom =
+    Form.useWatch(['resourceGroup'], {
+      form,
+      preserve: true,
+    }) || form.getFieldValue('resourceGroup');
```

Comment:
I'm not sure if this is the intended logic, but backend.ai currently allows an empty string for the resource group name.

Reply:
An empty string is different from a string containing a single whitespace (" "). 😉

Reply:
When I first realized the issue, I was only reporting on empty strings. 
Later, I realized that there were more to consider, such as characters starting with a space or containing a single whitespace, so I created an issue and worked on adding regular expressions. Thank you for notice! :)